recaptcha_enterprise/demosite/src/test/java/Util.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package app;
+
+import com.google.cloud.recaptchaenterprise.v1.RecaptchaEnterpriseServiceClient;
+import com.google.recaptchaenterprise.v1.CreateKeyRequest;
+import com.google.recaptchaenterprise.v1.DeleteKeyRequest;
+import com.google.recaptchaenterprise.v1.Key;
+import com.google.recaptchaenterprise.v1.KeyName;
+import com.google.recaptchaenterprise.v1.ProjectName;
+import com.google.recaptchaenterprise.v1.WebKeySettings;
+import com.google.recaptchaenterprise.v1.WebKeySettings.IntegrationType;
+import java.io.IOException;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+
+public class Util {
+
+  /**
+   * Create reCAPTCHA Site key which binds a domain name to a unique key.
+   *
+   * @param projectID : Google Cloud Project ID.
+   * @param domainName : Specify the domain name in which the reCAPTCHA should be activated.
+   */
+  public static String createSiteKey(String projectID, String domainName, IntegrationType keyType) throws IOException {
+    try (RecaptchaEnterpriseServiceClient client = RecaptchaEnterpriseServiceClient.create()) {
+
+      // Set the type of reCAPTCHA to be displayed.
+      // For different types, see: https://cloud.google.com/recaptcha-enterprise/docs/keys
+      Key scoreKey =
+          Key.newBuilder()
+              .setDisplayName("recaptcha_demosite_test_key")
```

Comment:
Please consider adding a random number/id at the end to avoid collisions if running couple tests ins parallel. Also please consider adding test- prefix to make cleaning up easier by reaper type of tools.

Reply:
Done. Reaper must be configured to use display name in this case and not the key name, as we do not have any control over the generated key name.